Dockerfile

```diff
@@ -5,7 +5,8 @@ LABEL org.opencontainers.image.source = "https://github.com/neuro-inc/platform-e
 #Based on https://developers.redhat.com/blog/2019/08/14/best-practices-for-running-buildah-in-a-container/
 
 RUN echo -e max_parallel_downloads=10\\nfastestmirror=true >> /etc/dnf/dnf.conf && \
-    dnf install -y --exclude container-selinux podman buildah python3 make gcc python3-devel jq git && \
+    dnf install -y --exclude container-selinux podman buildah python3 make gcc python3.9 python3-devel jq git && \
```

Comment:
Installing python3.9 here since wasn't able to run podman on a newer version of fedora container, was getting some weird error 
```
$ podman info --debug
WARN[0000] "/" is not a shared mount, this could cause issues or missing mounts with rootless containers 
cannot clone: Operation not permitted
Error: cannot re-exec process
```
and didn't managed to resolve it..